website/components/article.js

```diff
@@ -1,8 +1,10 @@
 /** @jsx jsx */
 import { jsx, css } from "@emotion/core";
 import { Fragment } from "react";
-import { Index, items } from "../pages";
+import { Index } from "../pages";
```

Comment:
This might be unrelated to your change, but thoughts being it is odd that a component is importing a page.